sites/website/src/docs/community/engineering-process/release-planning.md

```diff
@@ -0,0 +1,61 @@
+---
+id: release-planning
+title: Release Planning
+sidebar_label: Release Planning
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/engineering/release-planning.md
+---
+
+In order to both provide features and fixes to our community, while not creating unnecessary breaking changes or churn, FAST has adopted the following release planning process and policies.
+
+## Quarterly Planning
+
+As mentioned in [Issue Management](https://www.fast.design/docs/engineering/issue-management), feature teams engage in a quarterly planning exercise. During this exercise engineers look at the big picture and make plans to address broader "themes" that we see emerging which can't be handled week-to-week. The quarterly planning is then used to shape the weekly planning sessions that happen over the following three months. 
+
+The process for identifying quarterly work consists of multiple phases. In each phase, we filter issues out until we end end up with the quarterly plan.
+
+- Filtering & Individual prioritization
+- Rough costing
+- Team review & Priority adjustment
+- Capacity planning
+- Define the cut line
+
+### Filtering
+
+At this stage all the issues are distributed to engineers by feature areas. Each engineering lead reviews all the issues within their feature area, and returns with a prioritized list.
+
+### Rough costing
```

Comment:
Some headings capitalize all names and others do not.  For example, "Rough costing" and "Quarterly Planning".  Should we consistently name headings?